lab_4_summarization_textrank/main.py

```diff
@@ -184,7 +264,24 @@ def add_edge(self, vertex1: Sentence, vertex2: Sentence) -> None:
         :param vertex2:
         :return:
         """
-        pass
+        check_type(vertex1, Sentence)
+        check_type(vertex2, Sentence)
+
+        if vertex1 == vertex2:
```

Comment:
we establish that sentences are equal based on the specific field

Reply:
fixed